src/content/pages/for-nerds.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Credits
+---
+
+import Panel from 'components/Panel';
+
+<Panel>
+
+## How is Mic Drop built?
+
+Mic Drop is a macOS application written _entirely_ in Swift and SwiftUI. It uses **Core Audio** to manage mute/volume state. Mic Drop requires Input Monitoring permissions to listen for global keyboard shortcuts—particularly double-tap, modifier-only shortcuts. ⌨️
```

Comment:
```suggestion
Mic Drop is a macOS application written _entirely_ in Swift and SwiftUI. It uses Core Audio to manage mute and volume states. Mic Drop requires Input Monitoring permissions to listen for global keyboard shortcuts—particularly double-tap, modifier-only shortcuts. ⌨️
```

....why "particularly"?

Reply:
Because not using those shortcuts you can get away with not asking for permissions 😆